src/vs/editor/common/services/modelServiceImpl.ts

```diff
@@ -283,7 +119,7 @@ export class ModelServiceImpl extends Disposable implements IModelService {
 	constructor(
 		@IMarkerService markerService: IMarkerService | null,
```

Comment:
I think `IMarkerService` is no longer needed in this class, I would remove it.

Reply:
`IMarkerService` is still being used by `ModelServiceImpl` to remove the markers of models with internal URI scheme. I have no issues to move that clean up to new service and remove `IMarkerService` in `ModelServiceImpl`.

Reply:
Done